src/plugins/shared_ux/public/components/page_template/no_data_page/no_data_card/no_data_card.stories.tsx

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React from 'react';
+import { NoDataCard } from './no_data_card';
+import type { NoDataCardProps } from './types';
+
+export default {
+  title: 'No Data Card',
+  description: 'A wrapper around EuiCard, to be used on NoData page',
+};
+
+type Params = Pick<NoDataCardProps, 'recommended' | 'button' | 'description'>;
+
+export const PureComponent = (params: Params) => {
+  return (
+    <div style={{ width: '50%' }}>
+      <NoDataCard title={'You have no data'} {...params} />
```

Comment:
Sorry, one last nit pick here because I forgot that the default value for the button is to use the `title`. The way we originally designed it is that the title is action-oriented, so I'd just like the example to be the same.
```suggestion
      <NoDataCard title={'Add data'} {...params} />
```